src/sentry/migrations/0179_copy_useroptions_to_notificationsettings.py

```diff
@@ -0,0 +1,129 @@
+# Generated by Django 1.11.29 on 2021-03-15 17:28
+
+from django.db import migrations
+from sentry.models import UserOptionValue
+from sentry.models.actor import ACTOR_TYPES
+from sentry.models.integration import ExternalProviders
+from sentry.models.notificationsetting import (
+    NotificationScopeType,
+    NotificationSettingTypes,
+    NotificationSettingOptionValues,
+)
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def copy_useroption_to_notificationsetting(apps, schema_editor):
+    UserOption = apps.get_model("sentry", "UserOption")
+    NotificationSetting = apps.get_model("sentry", "NotificationSetting")
+    for user_option in RangeQuerySetWrapperWithProgressBar(UserOption.objects.all()):
+        if user_option.key == "workflow:notifications":
+            # if you have fine tuning for projects, project_id will have a value (rather than None)
+            if user_option.project_id:
+                scope_identifier = user_option.project_id
+                scope_type = NotificationScopeType.PROJECT
+            else:
+                scope_identifier = user_option.user.id
+                scope_type = NotificationScopeType.USER
+            type = NotificationSettingTypes.WORKFLOW
+            # value ###
+            # all_conversations = "0"
+            # participating_only = "1"
+            # no_conversations = "2"
+            if user_option.value == -1:
```

Comment:
I don't think the DB has any instances of "-1". Also we should prefer to have an empty row than value = `NotificationSettingOptionValues.DEFAULT`.

Reply:
cool, did some local poking around to confirm cause I'm paranoid and updated :)